saleor/graphql/shop/mutations.py

```diff
@@ -288,3 +288,125 @@ def perform_mutation(cls, _root, info, key_type):
 
         instance.delete()
         return AuthorizationKeyDelete(authorization_key=instance, shop=Shop())
+
+
+class StaffNotificationInput(graphene.InputObjectType):
+    staff_email = graphene.String(
+        required=False,
+        description="Staff email (instead of providinf staff member user).",
+    )
+    active = graphene.Boolean(required=False, description="Client secret.")
+    user = graphene.ID(required=False, description="The ID of the user to assign.")
+
+
+class StaffNotificationAdd(BaseMutation):
```

Comment:
`StaffNotificationCreate`?  I haven't seen any mutation with `Add` so I assume the convention is to use `Create`

Reply:
Have you tried to use ModelMutation? Wouldn't it be a better way?

Reply:
will check and fix